content/docs/tasks/security/authn-policy/index.md

```diff
@@ -1,90 +1,85 @@
-    sleep.bar to httpbin.foo: 200
-    sleep.bar to httpbin.bar: 200
-    sleep.bar to httpbin.legacy: 200
-    sleep.legacy to httpbin.foo: 200
-    sleep.legacy to httpbin.bar: 200
-    sleep.legacy to httpbin.legacy: 200
-    {{< /text >}}
+This one-liner command conveniently iterates through all reachability combinations:
 
-    > If you have `curl` installed on istio-proxy container, you can also verify that it can reach `httpbin` services in from proxy:
+{{< text bash >}}
+$ for from in "foo" "bar" "legacy"; do for to in "foo" "bar" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.${to}:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
+sleep.foo to httpbin.foo: 200
+sleep.foo to httpbin.bar: 200
+sleep.foo to httpbin.legacy: 200
+sleep.bar to httpbin.foo: 200
+sleep.bar to httpbin.bar: 200
+sleep.bar to httpbin.legacy: 200
+sleep.legacy to httpbin.foo: 200
+sleep.legacy to httpbin.bar: 200
+sleep.legacy to httpbin.legacy: 200
+{{< /text >}}
 
-    {{< text bash >}}
-    $ kubectl exec $(kubectl get pod -l app=sleep -n bar -o jsonpath={.items..metadata.name}) -c istio-proxy -n bar -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
-    200
-    {{< /text >}}
+You should also verify that there are no existing authentication policies in the system, which you can do as follows:
 
-* Last but not least, verify that there are no authentication policy:
+{{< text bash >}}
+$ kubectl get policies.authentication.istio.io --all-namespaces
+No resources found.
+{{< /text >}}
 
-    {{< text bash >}}
-    $ kubectl get meshpolicies.authentication.istio.io
-    No resources found.
-    {{< /text >}}
+{{< text bash >}}
+$ kubectl get meshpolicies.authentication.istio.io
+No resources found.
+{{< /text >}}
 
-    {{< text bash >}}
-    $ kubectl get policies.authentication.istio.io --all-namespaces
-    No resources found.
-    {{< /text >}}
+Last but not least, verify that there are no destination rules that apply on our example services. It can be done by checking the `host:` value of existing destination rules and make sure they do not match. For example:
 
-    {{< text bash >}}
-    $ kubectl get destinationrules.networking.istio.io --all-namespaces
-    {{< /text >}}
+{{< text bash >}}
+$ kubectl get destinationrules.networking.istio.io --all-namespaces -o yaml | grep "host:"
+    host: istio-policy.istio-system.svc.cluster.local
+    host: istio-telemetry.istio-system.svc.cluster.local
+{{< /text >}}
 
-    > You may see some policies and/or destination rules added by Istio installation, depends on installation mode. However, there should be none for `foo`, `bar` and `legacy` namespaces.
+> Depending on Istio version, you may see destination rules for hosts other than these. However, there should be none with host in `foo`, `bar`, `legacy` namespace, nor is the match-all wildcard `*`
```

Comment:
Depending on the version of Istio, you may see destination rules for hosts other then those shown.
with host -> with hosts